.github/workflows/go.yml

```diff
@@ -36,5 +36,5 @@ jobs:
     - name: Run staticcheck
       run: |
         cd yb-voyager
-        go install honnef.co/go/tools/cmd/staticcheck@latest
+        go install honnef.co/go/tools/cmd/staticcheck@2023.1.7
```

Comment:
just to confirm: this is second latest release?

Reply:
Right, excluding RC release of latest